src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/CreateApprenticeshipParametersMapper.cs

```diff
@@ -87,5 +114,18 @@ public CreateApprenticeshipParametersMapper(IWageTypeMapper wageTypeMapper, IDur
                 parameters.Postcode = employerInformation.Postcode;
             }
         }
+
+        private static void MapTrainingCode(CreateApprenticeshipRequest request, CreateApprenticeshipParameters parameters)
+        {
+            if (request.TrainingType == TrainingType.Standard)
+            {
+                parameters.TrainingCode = request.TrainingCode;
+            }
+            else
+            {
+                var length = request.TrainingCode.IndexOf("-", StringComparison.Ordinal);
```

Comment:
For frameworks, I think this will always be first 3 characters. Don't think there will be 4 and if you are trimming before this method gets called, you don't need to do index/length business so long as the string you are operating on is more than or equal to 3 characters..

Reply:
The length is left up to 4 to allow any future inclusions, hence not restricting to minimum.